packages/taquito-beacon-wallet/src/taquito-beacon-wallet.ts

```diff
@@ -39,14 +37,9 @@ export { BeaconWalletNotInitialized, MissingRequiredScopes } from './errors';
 
 export class BeaconWallet implements WalletProvider {
   public client: DAppClient;
-  public account: AccountInfo | undefined;
```

Comment:
Because account was public, removing it is a breaking change. We should consider mentioning it clearly in the release note if there is no other solution than removing it

Reply:
Great suggestion added to the pr description will make sure mentioning in the release note. Thank you!!